tests/checkbox/operate-checkbox-interaction.html

```diff
@@ -14,8 +14,7 @@
     task: "operate checkbox",
     specific_user_instruction: "check and uncheck the first checkbox",
     output_assertions: [
-      [1, "The change in the state of the checkbox is conveyed when the box becomes unchecked."],
-      [1, "The change in the state of the checkbox is conveyed when the box becomes checked."]
+      [1, "Change in state is conveyed"]
```

Comment:
I know we talked about this in the CG meeting, but was this for sure the conclusion? What if the state is only not convey when you go one direction? Or is that so unlikely, we shouldn't worry about it?

Reply:
I share the same concern, however it would be rare enough that a tester could describe which value failed in the notes.

Reply:
No need to bike shed so lets just go with this and change the test if it is insufficient :)